packages/grid/_modules_/grid/hooks/features/keyboard/useGridKeyboard.ts

```diff
@@ -61,9 +60,8 @@ export const useGridKeyboard = (
       }
 
       forceUpdate();
-      apiRef.current.publishEvent(GRID_MULTIPLE_KEY_PRESS_CHANGED, isPressed);
```

Comment:
what's the point of keeping `setMultipleKeyState` if you don't publish the event?

Reply:
It seems to update the gridState, which is used somewhere by another feature.

Reply:
Well that would involve refactoring the selection and the expand selection with the keyboard, which could move to the useSelection hook 🤔 

I suggest leaving this code as it is. As it might create an issue to remove the publication of the event at this stage
Do another PR to refactor the selection.